libs/full/compute_cuda/include/hpx/compute/cuda/detail/launch.hpp

```diff
@@ -109,6 +107,7 @@ namespace hpx { namespace cuda { namespace experimental { namespace detail {
                     std::string("kernel launch failed: ") +
                         cudaGetErrorString(error));
             }
+            HPX_UNUSED(launcher);
```

Comment:
What's the reasoning to duplicate this in both branches?

Reply:
Cause in one of them it's used, I was finding weird that we declared it unused and then used it ^^ but I don't mind reverting the change either

Reply:
Ah, makes sense. I missed that there are three branches.